samples/go/nomdex/vrange.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"sort"
+
+	"github.com/attic-labs/noms/go/types"
+)
+
+type ventry struct {
+	v    types.Value
+	incl bool
+	inf  int8
```

Comment:
But I can see the value in the way you have it. If you want to keep it that way, it is fine with me so long as you add a comment.